packages/medusa/src/api/routes/admin/products/get-product.ts

```diff
@@ -76,7 +75,7 @@ export default async (req, res) => {
   )
 
   let rawProduct
-  if (featureFlagRouter.isFeatureEnabled(IsolateProductDomainFeatureFlag.key)) {
+  if (featureFlagRouter.isFeatureEnabled(MedusaV2Flag.key)) {
```

Comment:
question: This slightly change the current behaviour. What we wanted to achieve if I am right is if the module is isolated with its own database. In that case we have to use the remoteQuery to target the module and retrieve the data. But it the user use the module in shared mode, then all the data are in the core database and the product service can be used. Here we are considering that no matter if it is shared or isolated we are going through the module. Now, that will work, but just wanted to point this behaviour changes just in case :D

It is probably the case in other places

Reply:
Just realised that maybe when the flag is enabled we don't really care if the modules are shared or isolated and we take the worst case, being isolated and it should work still. right?

Reply:
Yes, thats correct. 🤞🏻 

Shared or Isolated is just a resource sharing concern, it shouldn't be an issue on the app logic.